background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    # Post processing
</pre><p>I think it can be good, on the contrary, to have <code>GTensorToImg.on(g_nn_out)</code> used on its own, to show separately how the custom kernel was used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>